src/gevent/queue.py

```diff
@@ -372,9 +372,11 @@ class PriorityQueue(Queue):
     Entries are typically tuples of the form: ``(priority number, data)``.
     '''
 
-    def _init(self, maxsize, items=None):
+    def _init(self, maxsize, items=None, heapify=heapq.heapify):
+        # pylint:disable=arguments-differ
```

Comment:
I know that the sibling methods `_put` and `_get` do the performance hack of copying a variable into a kwarg, and I appreciate your being consistent. 

In this case, though, I don't think it's warranted. Can you remove the kwarg and thus fix (instead of ignore) the warning?